packages/sso-button-react/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,23 @@
+import GoogleIcon from '../assets/google.svg?react'
```

Comment:
in case your are wondering what `?react` is for.
https://www.npmjs.com/package/vite-plugin-svgr#usage